src/components/layout/TopBar.jsx

```diff
@@ -93,6 +94,32 @@ class TopBar extends Component {
                 </React.Fragment>
               )
             }
+            {
+              this.props.user.username !== undefined && this.props.useProfileDropdown === true
+              &&
+              (
+                <Popover
+                  title={this.props.user.username}
+                  placement='bottom'
+                  content={
+                    <React.Fragment>
+                      <Link to='/identity'>View Profile</Link>
+                      <br/>
+                      <Link to='#' onClick={this.props.onLogoutClick}>Logout</Link>
+                    </React.Fragment>
+                  }
+                >
+                  <Link className='top-bar__link' to='#'>
+                    <TopIconButton
+                      icon='user-circle'
+                      name=''
+                      isActive={this.isActive('/identity')}
+                      onActiveTab={() => this.props.onActiveTab('/identity')}
+                    />
+                  </Link>
```

Comment:
I thought for this part there were some inconsistence in the design. Have we made a dicision that whether we display only a single icon or icon + username after user has logged in

If we decide to keep only a single icon, can you adjust the `placement` of the popover so the arrow gets pointed to the icon?

Reply:
Either option is feasible, we'll have to put that one up to the team. Pushing a commit with correct `placement` now, that was just a temporary change that slipped through during experimentation.